labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -65,7 +94,7 @@ object CollectionExercise03 {
    * checkValuesIncrease(Seq(1,2,2)) == false
    */
   def checkValuesIncrease[T <% Ordered[T]](seq: Seq[T]): Boolean =
-    error("fix me")
+    !seq.indices.tail.map((i: Int) => (seq(i) > seq(i-1))).contains(false)
 
 }
```

Comment:
In all of my searching, I never encountered the indices method. Seems like a good thing to know about for this assignment and the future!